EIPS/eip-4401.md

```diff
@@ -0,0 +1,115 @@
+---
+eip: 4401
+title: EIP-721 Consumable Extension - Errata
+description: Interface extension for EIP-721 consumer role
+author: Daniel Ivanov (@Daniel-K-Ivanov), George Spasov (@Perseverance), Pandapip1 (@Pandapip1)
```

Comment:
@Daniel-K-Ivanov @Perseverance I would like to double check you're okay with me adding myself as an author here.